### ISSUE TITLE
[KSP 1.11.1] Corvus nosecone - symmetry axis is busted

### ISSUE THREAD

**Issue by captain828** [mwerle/Corvus_CF#9]
![image](https://user-images.githubusercontent.com/17725004/103714092-7b8eb100-4fc6-11eb-92e3-33bf4507cd03.png)

The symmetry axis in either symmetry modes is not behaving normally for the nose cone.
Object placement on the nose cone in general is weird.

All other parts are functioning just fine.

Let me know if you need further details.

**Comment by mwerle** [comment]
Hmm, I wonder whether this is the reason for the custom drag cube which I removed a couple of versions ago...  TBH I don't think I ever tried attaching anything to the nose cone before as it contains most everything already.

Will take a look, thanks.

**Comment by captain828** [comment]
Yeah, I only found it out by mistake not any real intention to attach stuff there.
I noticed there is a small compartment on the troubled side of the nose cone, maybe something to do with that?

I also found another weird attachment issue on the 1.875m adapter's light beige separator line which allows you to place stuff on it at weird angles. No clue if in KSP you can define a different "attachment mesh" compared to the model's 3D mesh but I'm guessing this is the issue here as this separator line is embossed on the model mesh.

**Comment by mwerle** [comment]
So I can confirm both those issues. 

In the case of the nose cone it seems to be that the origin for mirrored parts is on the edge of the part and not the centre.  There doesn't seem to be anything in the config which can fix this so it must be in the model itself. 

Unfortunately I don't have the source models so I don't think I can fix this. I know there's an importer for Blender but I'm also not a modeller so not sure how to go about reverse engineering the model.

If anybody could jump in and help that would be awesome!

**Comment by mwerle** [comment]
Importer : https://github.com/taniwha/io_object_mu

**Comment by captain828** [comment]
I'll have a look tomorrow to see if I can figure this out.

**Comment by mwerle** [comment]
I imported the nose cone and my assumption was correct - the nose cone's origin is offset from the world origin.

Will try to fix and re-export and see if it still works.  Along with this another long-standing modelling bug is the IVA cutout overlay for the pod itself. But I'm not really a modeller so struggle to fix this sort of stuff.

**Comment by mwerle** [comment]
Ok, I managed to adjust and export the nose cone (resulting file is MUCH bigger than the original though and the thumbnail is incorrect) and symmetry is fixed in the VAB.

HOWEVER, this also means that the capsule no longer tilts to the side when the parachute is deployed which looks very odd now. So there was actually some method to the madness of offsetting the origin of the nosecone...

I've no idea how to resolve this at this stage; needs more investigation. Might just be something we need to live with.

**Comment by captain828** [comment]
Ah, I see what the intent was now.
Essentially to recreate this condition:

> At an altitude of roughly 15,000 meters the astronauts would deploy a 2.4 meter drogue chute from the rendezvous and recovery section. 
At 3230 meters altitude the crew releases the drogue which extracts the 5.5 meter pilot parachute. 
The rendezvous and recovery section is released 2.5 seconds later, deploying the 25.6 meter main ring-sail parachute which is stored in the bottom of the section. 
**The spacecraft is then rotated from a nose-up to a 35 degree angle for water landing.** At this point a recovery beacon is activated, transmitting via an HF whip antenna mounted near the front of the reentry module.
>
> [Gemini 8 NASA report, the section on Reentry ](https://nssdc.gsfc.nasa.gov/nmc/spacecraft/display.action?id=1966-020A)

From what I understand, the nose cone would have the drogue chute + a secondary and the command module would have the main.
The nose cone (or rendezvous and recovery section) would be decoupled before the 35-degree angle they mention is set, however.

This was likely a compromise they had to make as you can't tell KSP to not be able to deploy a chute because another thing is attached to it (unless modded in?). This likely then led to not being able to have the 35-degree angle anymore due to the extra weight of the nose cone attached, moving the CoM up, hence "needing" the offset.

At most, we should check to make sure the parachute is coming off the side of the nose cone and not from the center.
The bad origin should definitely not be a thing as this can have unexpected effects, like the drag you mentioned or tilting the CoM ever so slightly.

> long-standing modelling bug is the IVA cutout overlay for the pod itself.

What's the issue there?

**Comment by mwerle** [comment]
So the part configuration file in the original adjusts for the model offset by moving the node attachment points and the CoM appropriately.

The parachute does deploy off-centre from one side of the nosecone (after jettisoning a cover).

Attached is the "fixed" nose cone and a craft to test both of them side by side. Put the nose cone into the Parts folder for Corvus, and the craft into your savegame / Ships / VAB folder, load and launch. 

The "fixed" nosecone doesn't tilt the craft on descent which also increases the descent speed (less aerodynamic forces).

[Corvus_Nose_New.zip](https://github.com/mwerle/Corvus_CF/files/5782198/Corvus_Nose_New.zip)


If you have any suggestions on how to fix this that would be awesome!

As for the IVA, it's not the IVA itself that's broken but the external cutout view. It needs an additional layer or something (I forget the details; I was playing with it ages ago) for the external cutout to look decent.

**Comment by captain828** [comment]
Had a look at the old .mu vs the new one + the configs and I have a few things to mention but it would probably be easier and faster if we connected on discord.
My id is captain828#0846

**Comment by mwerle** [comment]
> Had a look at the old .mu vs the new one + the configs and I have a few things to mention but it would probably be easier and faster if we connected on discord.
> My id is captain828#0846

I am but I have my doubts - firstly I'm away on a business trip and secondly it depends on timezones. Lastly it means the information isn't captured and tracked.  That said, I'll ping you there and see how it goes.

PS. I'm Myshka#1601

**Comment by captain828** [comment]
No worries.
I will be sure to write it down and add it here once we're done.
